lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -307,6 +313,110 @@ void FunctionalChaosResult::setInvolvesModelSelection(const Bool involvesModelSe
   involvesModelSelection_ = involvesModelSelection;
 }
 
+/* Conditional expectation accessor */
+FunctionalChaosResult FunctionalChaosResult::getConditionalExpectation(const Indices & conditioningIndices) const
+{
+  // Get marginal input sample and distribution
+  const UnsignedInteger inputDimension = inputSample_.getDimension();
+  conditioningIndices.check(inputDimension);
```

Comment:
check returns a bool and you must throw an exception if false

Reply:
You are right: I have to fix this.

Reply:
This is fixed in 271a037da58754890130951f83d7f7b33635a80c